unit_tests/util/query_heap.cpp

```diff
@@ -22,9 +22,8 @@ struct TestData
 using TestNodeID = NodeID;
 using TestKey = int;
 using TestWeight = int;
-using storage_types = boost::mpl::list<ArrayStorage<TestNodeID, TestKey>,
-                                       MapStorage<TestNodeID, TestKey>,
-                                       UnorderedMapStorage<TestNodeID, TestKey>>;
+using storage_types =
```

Comment:
mpl::list is a fancy tuple, isn't it?

Reply:
It seems not exactly... Tuple is runtime type, but mpl is all about meta programming, i.e. it is list of types. It seems MPL is required to use `BOOST_FIXTURE_TEST_CASE_TEMPLATE` we are using below. 
I couldn't find official docs for this macro, but at least this SO example uses it(and it is not obvious how could I substitute it to smth else):
https://stackoverflow.com/questions/22065225/execute-one-test-case-several-times-with-different-fixture-each-time